### PR TITLE
BM-179 refine npm deployment files

### PR DESCRIPTION
**Pull request by EyesOnlyNet**
* add gcs provider to travis

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,13 +22,9 @@
...
     "UBP",
-    "CRS",
-    "travel",
-    "toma",
-    "cets",
-    "sabre"
+    "CRS"
   ],
-  "author": "Steve Reichenbach",
+  "author": "FTI",
```
Comment (tarleb) [inline package.json]: Well, technically, you are still the author.

File: prepare-gcs.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+// require
+var fs = require('fs');
+var pckg = require('./package.json');
+
+// helper
+var copy = function(srcDir, dstDir) {
```
Comment (tarleb) [inline prepare-gcs.js]: Why not just use the [copy](https://www.npmjs.com/package/copy) package?

File: prepare-gcs.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                fs.writeFileSync(dst, fs.readFileSync(src));
+            } catch(e) {
+                console.log('could\'t copy file: ' + dst);
+            }
+            results.push(src);
+        }
+    });
+    return results;
+};
+var deleteFolderRecursive = function(path) {
```
Comment (tarleb) [inline prepare-gcs.js]: I suspect rimraf would be overkill?

File: package.json (hunk 1 above)
Comment (EyesOnlyNet) [reply package.json]: Yes, but I saw it on another repo of tourstream and thought: "that could be more correct".

File: prepare-gcs.js (hunk 2 above)
Comment (EyesOnlyNet) [reply prepare-gcs.js]: I copied it only from another repo from us: https://github.com/tourstream/fti-portal-styleguide/blob/master/prepare-gcs.js

@dasrick What do you think of the idea, to create an own npm library for that?

**Pull request merged**